ext/cuda/operators_spectral_element.jl

```diff
@@ -269,14 +268,53 @@ Base.@propagate_inbounds function operator_evaluate(
     Nq = Quadratures.degrees_of_freedom(QS)
     D = Quadratures.differentiation_matrix(FT, QS)
 
-    ∂f∂ξ₁ = D[i, 1] * input[1, j, vt]
-    ∂f∂ξ₂ = D[j, 1] * input[i, 1, vt]
-
-    for k in 2:Nq
-        ∂f∂ξ₁ += D[i, k] * input[k, j, vt]
-        ∂f∂ξ₂ += D[j, k] * input[i, k, vt]
+    if length(input) == 1 # check types
+        (v₁,) = input
+        ∂f∂ξ₁ = D[i, 1] ⊠ v₁[1, j, vt]
+        ∂f∂ξ₂ = D[j, 1] ⊠ v₁[i, 1, vt]
+        for k in 2:Nq
```

Comment:
```suggestion
        @inbounds for k in 2:Nq
```

Reply:
Actually, maybe we can just wrap the whole block in `@inbounds` since the initial values computed can use them too

Reply:
Updated to wrap blocks in `@inbounds` ; not sure why the formatter now fails since there are no changes to the `DataLayouts` tests in this PR